.github/actions/delete-deployments/action.yml

```diff
@@ -0,0 +1,56 @@
+name: delete-deployments
+
+description: Deletes the deployments created by a workflow run in the scope of the PR.
```

Comment:
There's nothing in the logic limiting the scope to PRs: this deletes all deployments on a given branch

Reply:
Modified the description.